fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2352,6 +2354,26 @@ class DWALPager final : public IPager2 {
 			wait(self->commit());
 		}
 
+		if (!self->memoryOnly) {
+			wait(store(fileSize, self->pageFile->size()));
+		}
+
+		TraceEvent e(SevInfo, "RedwoodRecoveredPager");
+		e.detail("FileName", self->filename.c_str());
+		e.detail("LogicalFileSize", self->pHeader->pageCount * self->physicalPageSize);
+		e.detail("PhysicalFileSize", fileSize);
+		e.detail("OpenedExisting", exists);
+		e.detail("CommittedVersion", self->pHeader->committedVersion);
+		e.detail("LogicalPageSize", self->logicalPageSize);
+		e.detail("PhysicalPageSize", self->physicalPageSize);
+
+		self->remapQueue.toTraceEvent(e, "RemapQueue");
+		self->delayedFreeList.toTraceEvent(e, "FreeQueue");
+		self->freeList.toTraceEvent(e, "DelayedFreeQueue");
+		self->extentUsedList.toTraceEvent(e, "UsedExtentQueue");
+		self->extentFreeList.toTraceEvent(e, "FreeExtentQueue");
+		self->getStorageBytes().toTraceEvent(e);
```

Comment:
It doesn't seem to matter here now, but to possibly avoid confusion in the future you could run `e.log()` here so that it logs immediately rather than when the function returns.

Reply:
I didn't do that here because the function is already about to return, but I like the idea and it should probably be the best practice for named `TraceEvent` instances.